src/index.js

```diff
@@ -26,6 +26,7 @@ function splitGuide({
     .then(readAllFilesAsPromise)
     .then(createNewFileContents)
     .then(saveFiles)
+    .then(reduceFilenames)
```

Comment:
What was the purpose of adding this? I don't think I mind it, but I'm curious why you added it :)